app.js

```diff
@@ -75,8 +71,50 @@ CookieStand.prototype.cookieStandTableCells = function() {
   cookieTable.appendChild(trEl);
 };
 
-pike.cookieStandTableCells();
-seaTac.cookieStandTableCells();
-seaCent.cookieStandTableCells();
-capHill.cookieStandTableCells();
-alki.cookieStandTableCells();
+// render the header and table cells with conttent
+var render = function() {
+  makeHeaderCells();
+  for(var i = 0; i < stands.length; i++) {
+    stands[i].cookieStandTableCells();
+  }
+};
+
+render();
+// Grab form in html
+var cookieForm = document.getElementById('cookie_form');
+// var cookieFormArray = [];
+
+// event handler
```

Comment:
Try to avoid excessive commenting when pushing code for submission. Think of it like you're putting out a product. Commenting on a few chunk pieces of functionality makes sense and is fine. A comment like this that identifies an event handler is unnecessary because guess what? The next line takes a parameter named `event` and the function is called `handleCookieForm`. One could assume that that is the event handler.